inst/include/dust/gpu/kernels.hpp

```diff
@@ -135,6 +136,7 @@ void run_particles(size_t step_start,
                            shared_real,
                            nullptr,
                            false,
+                           false,
```

Comment:
Helpful to set these bools to named variables to be passed, as it's getting harder to interpret what they mean?